src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiItemSelection/Item.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import type {Node} from 'react';
+import {SortableHandle} from 'react-sortable-hoc';
+import Icon from '../Icon';
+import itemStyles from './item.scss';
+
+const DRAG_ICON = 'ellipsis-v';
+const REMOVE_ICON = 'trash';
+
+type Props = {
+    id: string | number,
+    index: number,
+    children: Node,
+    onRemove: (id: string | number) => void,
```

Comment:
I think the `onRemove` should be optional, and the icon should only be shown if it is passed. That's probably needed for the smart content, where you can't remove a row like that.